.github/workflows/dump-ci-stats.yml

```diff
@@ -0,0 +1,42 @@
+name: Dump GH CI Stats
+
+on:
+  workflow_run: # This allows the workflow to be reused
+    workflows:
+      [
+        'Integration Tests',
+        'Test Golang',
+        'golangci-lint',
+        'Build release Docker Images',
+        'Test all Packages',
+        'Test Documentation',
+        'Manage integration check',
+        'after-merge.yml',
+      ]
+    types:
+      - completed
+
+jobs:
+  final_job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Clear npm cache
+        run: npm cache clean --force
```

Comment:
To avoid corrupted or stale cache issue. Encountered this issue hence clearing the cache. Also its one package installation no such major impact.